src/data-storage/insert-trade.ts

```diff
@@ -0,0 +1,29 @@
+import { PrismaClient, Trade } from '@prisma/client';
+import { CreateTradeInput } from '../types';
+
+export const insertTrade = async (trade: CreateTradeInput) => {
+  const prisma = new PrismaClient();
+  try {
+    await prisma.trade.create({
+      data: {
+        id: trade.trade_id,
+        quantity: parseFloat(trade.last_size),
+        time: new Date(trade.time),
+        action: trade.side === 'buy' ? 'BUY' : 'SELL',

```

Comment:
Flavor: As `trade.side` is only ever `buy` or `sell`, if we are looking to store it as an uppercase enum, we can just use `trade.side.toUppercase()`